README.md

```diff
@@ -52,6 +53,16 @@ module SalesforceArSync::SalesforceSync
 end
 ````
 
+### actionpack-xml_parser
+Rails 4.0 removed support for parsing XML parameters form the request. However, the salesforce_ar_sync gem depends on this for handling Outbound Messages from Salesforce. If you are using salesforce_ar_sync with Rails >= 4.0, you will need to add the [actionpack-xml_parser](https://github.com/rails/actionpack-xml_parser) gem to your Gemfile:
+
+`gem 'actionpack-xml_parser'`
```

Comment:
If this is required for operation on rails 4 then shouldn't we put it as part of our dependancies?  Since this version of the gem will require rails 4 anyway?